CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+## Developer Tool Prerequisites
+* [go](https://golang.org/doc/install) - The Go programming language
+* [dep](https://golang.github.io/dep/docs/installation.html) - Go dependency manager
+* [nvm](https://github.com/creationix/nvm#installation) - Node Version Manager (for installing NodeJS and NPM)
+
+## Performing a Fresh Checkout
+The following steps guide you through a fresh checkout
+
+```
+# Initalise a Go Workspace and checkout project sources
+go get -u github.com/improbable-eng/grpc-web/go
+cd src/github.com/improbable-eng/grpc-web
```

Comment:
This command is relative to `$GOPATH`, do we want to make it a little more explicit?

```
cd $GOPATH/src/...
```

Reply:
Better yet, I should probably set the `GOPATH` :)